src/content/samples/golang.mdx

```diff
@@ -13,7 +13,7 @@ This tutorial will show you how to develop and debug a Go sample application run
 
 ## Prerequisites
 
-- Install the Okteto CLI >= 1.9.0. Follow this [guide](/docs/cloud/okteto-cli/) if you haven't done it yet.
+- Install the Okteto CLI >= 2.0.0. Follow this [guide](/docs/cloud/okteto-cli/) if you haven't done it yet.
```

Comment:
I don't think is necessary to specify the version. Maybe just mention 'install the latest version of the Okteto CLI'?